docs/samples/Microsoft.ML.Samples/Program.cs

```diff
@@ -1,25 +1,29 @@
 using System;
 using System.Reflection;
+using System.Text.RegularExpressions;
 using Samples.Dynamic;
 
 namespace Microsoft.ML.Samples
 {
     public static class Program
     {
-        public static void Main(string[] args) => RunAll();
+        public static void Main(string[] args) => RunAll("LocalizeRootCauseMultipleDimensions");
```

Comment:
Ignore this. Just for running sample

Reply:
I think this is a good change. Worth keeping.  Except that the flow below should be modified to run all tests if name ==  null.

---
In reply to: [439605531](https://github.com/dotnet/machinelearning/pull/5236#discussion_r439605531) [](ancestors = 439605531)